src/05-derivatives/02-common-data-types.md

```diff
@@ -1,6 +1,6 @@
 # Common data types
 
-## Processed, coregistered and/or resampled volumes
+## Preprocessed or cleaned data
 
```

Comment:
I believe the template not only applies to `anat|func|dwi`, but also to `meg|eeg|ieeg`. This is included further down in the example as time-domain filtering.

Reply:
Should we give up the `anat|func|dwi|meg|...` here and go with `<datatype>/`? (I think that's the word we use for this level.) This really should apply to all.

Reply:
I think `<datatype>/` (or any similar language) would be best (thinking of `connectivity/` from BEP017)